install.sh

```diff
@@ -6,141 +6,142 @@ DIR="$( cd "$( dirname "$0" )" && pwd )"
+}
+
 # Find target dirs for various browsers & OS'es
 # https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location
 # https://wiki.mozilla.org/WebExtensions/Native_Messaging
 OPERATING_SYSTEM=$(uname -s)
 
 case $OPERATING_SYSTEM in
-Linux)
-  HOST_FILE="$DIR/browserpass-linux64"
-  if [ "$(whoami)" == "root" ]; then
-    TARGET_DIR_CHROME="/etc/opt/chrome/native-messaging-hosts"
-    TARGET_DIR_CHROMIUM="/etc/chromium/native-messaging-hosts"
-    TARGET_DIR_FIREFOX="/usr/lib/mozilla/native-messaging-hosts"
-    TARGET_DIR_VIVALDI="/etc/chromium/native-messaging-hosts"
-  else
-    TARGET_DIR_CHROME="$HOME/.config/google-chrome/NativeMessagingHosts"
-    TARGET_DIR_CHROMIUM="$HOME/.config/chromium/NativeMessagingHosts"
-    TARGET_DIR_FIREFOX="$HOME/.mozilla/native-messaging-hosts"
-    TARGET_DIR_VIVALDI="$HOME/.config/vivaldi/NativeMessagingHosts"
-  fi
-  ;;
-Darwin)
-  HOST_FILE="$DIR/browserpass-darwinx64"
-  if [ "$(whoami)" == "root" ]; then
-    TARGET_DIR_CHROME="/Library/Google/Chrome/NativeMessagingHosts"
-    TARGET_DIR_CHROMIUM="/Library/Application Support/Chromium/NativeMessagingHosts"
-    TARGET_DIR_FIREFOX="/Library/Application Support/Mozilla/NativeMessagingHosts"
-    TARGET_DIR_VIVALDI="/Library/Application Support/Vivaldi/NativeMessagingHosts"
-  else
-    TARGET_DIR_CHROME="$HOME/Library/Application Support/Google/Chrome/NativeMessagingHosts"
-    TARGET_DIR_CHROMIUM="$HOME/Library/Application Support/Chromium/NativeMessagingHosts"
-    TARGET_DIR_FIREFOX="$HOME/Library/Application Support/Mozilla/NativeMessagingHosts"
-    TARGET_DIR_VIVALDI="$HOME/Library/Application Support/Vivaldi/NativeMessagingHosts"
-  fi
-  ;;
-OpenBSD)
-  HOST_FILE="$DIR/browserpass-openbsd64"
-  if [ "$(whoami)" == "root" ]; then
-    echo "Installing as root not supported."
-    exit 1
-  fi
-  TARGET_DIR_CHROME="$HOME/.config/google-chrome/NativeMessagingHosts"
-  TARGET_DIR_CHROMIUM="$HOME/.config/chromium/NativeMessagingHosts"
-  TARGET_DIR_FIREFOX="$HOME/.mozilla/native-messaging-hosts"
-  TARGET_DIR_VIVALDI="$HOME/.config/vivaldi/NativeMessagingHosts"
-  ;;
-FreeBSD)
-  HOST_FILE="$DIR/browserpass-freebsd64"
-  if [ "$(whoami)" == "root" ]; then
-    echo "Installing as root not supported"
-    exit 1
-  fi
-  TARGET_DIR_CHROME="$HOME/.config/google-chrome/NativeMessagingHosts"
-  TARGET_DIR_CHROMIUM="$HOME/.config/chromium/NativeMessagingHosts"
-  TARGET_DIR_FIREFOX="$HOME/.mozilla/native-messaging-hosts"
-  TARGET_DIR_VIVALDI="$HOME/.config/vivaldi/NativeMessagingHosts"
-  ;;
-*)
-  echo "$OPERATING_SYSTEM is not supported"
-  exit 1
-  ;;
+    Linux)
+        HOST_FILE="$DIR/browserpass-linux64"
+        if [ "$(whoami)" == "root" ]; then
+            BROWSER_PATHS=( "Chrome:/etc/opt/chrome/native-messaging-hosts"
+            "Chromium:/etc/chromium/native-messaging-hosts"
+            "Firefox:/usr/lib/mozilla/native-messaging-hosts"
+            "Vivaldi:/etc/chromium/native-messaging-hosts" )
```

Comment:
Chromium and Vivaldi have the same folder, so Vivaldi is being falsely discovered as available. Do you think it is a copy-paste issue that nobody has discovered so far? 😄

Reply:
Wow, I hadn't actually noticed that. It's literally been there [forever](https://github.com/dannyvankooten/browserpass/commit/9a313be3ad53ecdfb4bc53dd1c2543af5570038f#diff-3fbb47e318cd8802bd325e7da9aaabe8R31) though 😄 

I'll follow up with a fix sometime today.